### PR TITLE
mrc-1544 auto-generate types

### PR DESCRIPTION
**Pull request by hillalex**
Almost exactly the same as hint, just using schemas from 2 places

### CODE REVIEW

**Comment by EmmaLRussell** [comment]
> Almost exactly the same as hint, just using schemas from 2 places

I guess we should also check responses from the API against the pkgapi schema in the backend integration tests. I'll add a ticket..

File: src/app/static/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+## MINT Front End
```
Comment (EmmaLRussell) [inline src/app/static/README.md:1]: Great yes, we were missing this file before. Might be nice to add a link to this from the top level README, I think we have that in HINT.

File: src/app/static/src/app/generated.d.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
+/**
+  * This file was automatically generated.
+  * DO NOT MODIFY IT BY HAND.
+  * Instead, modify the hintr JSON schema files
```
Comment (EmmaLRussell) [inline src/app/static/src/app/generated.d.ts:4]: Is this now slightly out of date? Ran locally and saw a couple of differences in the Response types as well as the corrected hintr->mintr message.

Comment (hillalex) [reply src/app/static/src/app/generated.d.ts:4]: There are some bugs in the pkgapi schemas so I did in fact edit this by hand! But yes I'll also manually fix the hintr -> mintr

Comment (hillalex) [reply src/app/static/src/app/generated.d.ts]: (PR for fixing the pkgapi schemas is in progress at the moment https://github.com/reside-ic/pkgapi/pull/10)

**Pull request merged**